Tools/TestWebKitAPI/Tests/WebKitCocoa/SOAuthorizationTests.mm

```diff
@@ -646,11 +667,15 @@ static void checkAuthorizationOptions(bool userActionInitiated, String initiator
 #endif
     EXPECT_FALSE(policyForAppSSOPerformed); // The delegate isn't registered, so this won't be set.
 
-    RetainPtr<NSURL> redirectURL = [[NSBundle mainBundle] URLForResource:@"simple2" withExtension:@"html" subdirectory:@"TestWebKitAPI.resources"];
-    auto response = adoptNS([[NSHTTPURLResponse alloc] initWithURL:testURL statusCode:302 HTTPVersion:@"HTTP/1.1" headerFields:@{ @"Location" : [redirectURL absoluteString] }]);
+    TestWebKitAPI::HTTPServer server({
+        { "/"_s, { SimpleHtml } },
+    }, TestWebKitAPI::HTTPServer::Protocol::Http, nullptr, nullptr, 8091);
```

Comment:
Why port 8091?  Is that just one you're guessing is available?  It's typically better to let the server pick one that is available and send requests to whichever one it picks.

Reply:
Done